libdeno/binding.cc

```diff
@@ -409,7 +412,15 @@ v8::MaybeLocal<v8::Module> ResolveCallback(v8::Local<v8::Context> context,
   v8::Context::Scope context_scope(context);
 
   int ref_id = referrer->GetIdentityHash();
-  std::string referrer_filename = d->module_filename_map_[ref_id];
+  auto range = d->module_info_map_.equal_range(ref_id);
+  std::string referrer_filename = "";
+  for (auto it = range.first; it != range.second; ++it) {
```

Comment:
If I understand this correctly, this loop should almost always end after the first iteration?

Reply:
Most of the times if there is no hash collision.